src/current/_includes/releases/v23.2/v23.2.5.md

```diff
@@ -0,0 +1,68 @@
+## v23.2.5
+
+Release Date: May 7, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-5-sql-language-changes">SQL language changes</h3>
+
+- The new [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.stats.virtual_computed_columns.enabled` enables collection of table statistics on virtual computed columns. [#120923][#120923]
```

Comment:
suggest directly linking to the cluster setting, they all have anchor links 

suggest adding links to table statistics [1] and computed columns [2]

[1]: https://www.cockroachlabs.com/docs/v23.2/cost-based-optimizer#table-statistics
[2]: https://www.cockroachlabs.com/docs/v23.2/computed-columns.html